public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -80,19 +80,34 @@ import { FeatureAccordion } from '../../../../public/pages/ConfigureModel/compon
 import {
   AD_DOCS_LINK,
   AD_HIGH_CARDINALITY_LINK,
+  ALERTING_PLUGIN_NAME,
+  DEFAULT_SHINGLE_SIZE,
   MAX_FEATURE_NUM,
 } from '../../../../public/utils/constants';
-import { getNotifications } from '../../../../public/services';
+import { getNotifications, getUiActions } from '../../../../public/services';
 import { prettifyErrorMessage } from '../../../../server/utils/helpers';
 import {
   ORIGIN_PLUGIN_VIS_LAYER,
   OVERLAY_ANOMALIES,
   VIS_LAYER_PLUGIN_TYPE,
 } from '../../../../public/expressions/constants';
 import { formikToDetectorName, visFeatureListToFormik } from './helpers';
+import { AssociateExisting } from './AssociateExisting';
+import { mountReactNode } from '../../../../../../src/core/public/utils';
+import { CoreServicesContext } from '../../../../public/components/CoreServices/CoreServices';
+import { CoreStart } from '../../../../../../src/core/public';
 
-function AddAnomalyDetector({ embeddable, closeFlyout, mode, setMode }) {
+function AddAnomalyDetector({
+  embeddable,
+  closeFlyout,
+  mode,
+  setMode,
+  selectedDetector,
+  setSelectedDetector,
+}) {
   const dispatch = useDispatch();
+  const core = React.useContext(CoreServicesContext) as CoreStart;
```

Comment:
I don't see core being used, i think it can be removed. Any sub-core services (like notifications) should be accessed via getters/setters in the services file.

Reply:
yeah just had it while testing